php/tower/app/Exceptions/MethodNotAllowedException.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
having exception logic should not be handled inside `towe` package ? I mean : do the develop need to define this class to handle http 405 ?